charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 19.2.2
+version: 20.0.0
```

Comment:
apologies for the delayed response. can you please fix the conflicts, I will approve this PR

Reply:
No worries, I also completely forgot about this PR. Just rebased it and bumped the version again.